src/beam3/4C_beam3_euler_bernoulli_evaluate.cpp

```diff
@@ -235,7 +235,10 @@ int Discret::Elements::Beam3eb::evaluate_neumann(Teuchos::ParameterList& params,
   // get element displacements
   std::shared_ptr<const Core::LinAlg::Vector<double>> disp =
       discretization.get_state("displacement new");
-  if (disp == nullptr) FOUR_C_THROW("Cannot get state vector 'displacement new'");
+  if (disp == nullptr)
+    FOUR_C_THROW(
+        "Cannot get state vector 'displacement new'. "
+        "No linearization provided for load (add 'LOADLIN: true' to input file)");
```

Comment:
This is a possible fix. But is it the actual and only reason? I just want to avoid, that the error message steers the user into the possibly wrong direction, if the actual reason is a different one.